Source/JavaScriptCore/bytecompiler/BytecodeGenerator.cpp

```diff
@@ -1713,8 +1713,24 @@ RegisterID* BytecodeGenerator::emitBinaryOp(OpcodeID opcodeID, RegisterID* dst,
         return emitBinaryOp<OpLess>(dst, src1, src2, types);
     case op_lesseq:
         return emitBinaryOp<OpLesseq>(dst, src1, src2, types);
-    case op_greater:
+    case op_greater: {
+        // We treat `typeof x > "u"` as `typeof x === "undefined"`
```

Comment:
I can't wait until there's a proposal for a new primitive whose type starts with `v` and it gets rejected because it makes everyone's code size bigger... lol